_sass/_basic.scss

```diff
@@ -38,9 +38,22 @@
     margin-bottom: 0;
 }
 
+.tla-descrip {
+    font-size: 5.5vw;
+
+    @media (min-width: $desktop-width) {
+        font-size: 1.5em;
+    }
+}
+
 .page-title{
-    font-size: 3em;
+    // font-size: 3em;
```

Comment:
Can you remove this comment? Other than that, looks good to merge!

Reply:
sorry! I'll do that. Do you still want the comments in variables.scss?

Reply:
Oh no those also aren't needed, I guess I forgot to delete them when I ported over our ACM stuff!